utilities/check_mrTrix.m

```diff
@@ -1,17 +1,38 @@
-function [status, mrTrixpaths] = check_mrTrix
```

Comment:
Put an argument check in here so that it checks mrtrix 2 by default if there is no input argument